transport/src/main/java/io/netty/channel/Channel.java

```diff
@@ -201,7 +201,6 @@
      *   <li>{@link #localAddress()}</li>
      *   <li>{@link #remoteAddress()}</li>
      *   <li>{@link #closeForcibly()}</li>
-     *   <li>{@link #register(EventLoop, ChannelPromise)}</li>
```

Comment:
@kezhenxu94 The method actually exists but changed the signature slightly. Please fix the arguments and not remove it here:

```
        void register(ChannelPromise promise);
```

Reply:
@normanmaurer Done